sdk/eventhub/azure-eventhub/azure/eventhub/exceptions.py

```diff
@@ -36,7 +38,7 @@ def _error_handler(error):
     return errors.ErrorAction(retry=True)
 
 
-class EventHubError(Exception):
+class EventHubError(AzureError):
```

Comment:
could you also kindly help update the changelog saying that:
```
**New features**
- `azure.eventhub.exceptions.EventHubError` now inherits from `azure.core.exceptions.AzureError`.
```

Reply:
@yunhaoling added in latest commit 👍